frontend/src/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,82 @@
+import ReactMarkdown from "react-markdown";
+import rehypeHighlight from "rehype-highlight";
+import "highlight.js/styles/atom-one-dark.css";
+import { Box, Flex, Text, useColorModeValue } from "@chakra-ui/react";
+import CopyButton from "./CopyButton";
+import { FiTerminal } from "react-icons/fi";
+import { v4 } from "uuid";
+
+const Markdown = ({ content }: { content: string }) => {
+  const textColor = useColorModeValue("ui.dark", "ui.white");
+  const secBgColor = useColorModeValue("ui.secondary", "ui.darkSlate");
+  return (
+    <ReactMarkdown
+      rehypePlugins={[rehypeHighlight]}
+      components={{
+        pre: ({ children }) => (
+          <Box as="pre" overflow="auto">
+            {children}
+          </Box>
+        ),
+        code: ({ node, className, children, ...props }) => {
+          const match = /language-(\w+)/.exec(className || "");
+          if (match?.length) {
+            const id = v4();
+            return (
+              <Box borderWidth="1px" borderRadius="md" overflow="hidden" my={2}>
+                <Flex
+                  align="center"
+                  justify="space-between"
+                  bg={secBgColor}
+                  p={1}
+                  borderBottomWidth="1px"
+                >
+                  <Flex align="center" gap={2}>
+                    <FiTerminal size={18} />
+                    <Text fontSize="sm" color={textColor}>
+                      {node?.data?.meta}
```

Comment:
Getting error: `Property 'meta' does not exist on type 'ElementData'.`
Maybe we can replace it with the language?
Example:
```js
const match = (className || "").match(/language-(\w+)/);
const language = match ? match[1] : null;
...
<Text fontSize="sm" color={textColor}>
{language}
</Text>
```

Reply:
Sorry for the inconvenience. I need more information, as I haven't encountered this error

Reply:
No worries! The build is failing as typescript is complaining that `node.data`'s `ElementData` type does not have a property called `meta`. You can check out the error logs in the last [test job](https://github.com/StreetLamb/tribe/actions/runs/10535248716/job/29197100710).

Reply:
Thans，recently discovered this issue and have updated it to display the name of the programming language instead. it is WIP
![image](https://github.com/user-attachments/assets/d7207277-62c5-4462-a627-2583370162c7)

Reply:
Thank you for the great suggestion; I've made the changes and updated the theme to use 'highlight.js/styles/github.css' for light mode to match the dark mode theme